src/SimpleChocolateyServer/Web.config

```diff
@@ -73,7 +73,7 @@
     </handlers>
     <staticContent>
       <remove fileExtension=".ps1" />
-      <mimeMap fileExtension=".nupkg" mimeType="application/zip"/>
+      <mimeMap fileExtension=".nupkg" mimeType="application/zip" />

```

Comment:
These are formatting changes - they go in a separate maint commit or are undone. The only time formatting changes should be with other changes is when it is unavoidable or part of the subject being changed.

Reply:
Argh, never spotted that. Will rebase and fix that.

Reply:
No worries here.